src/Context/Propagation/MultiTextMapPropagator.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Context\Propagation;
+
+use function array_map;
+use function array_merge;
+use function array_unique;
+use OpenTelemetry\Context\Context;
+
+final class MultiTextMapPropagator implements TextMapPropagatorInterface
+{
+    /**
+     * @readonly
+     *
+     * @var list<TextMapPropagatorInterface>
+     */
+    private array $propagators = [];
+
+    /**
+     * @readonly
+     *
+     * @var list<string>
+     */
+    private array $fields;
+
+    /**
+     * @no-named-arguments
+     *
+     * @param list<TextMapPropagatorInterface> $propagators
+     */
+    public function __construct(array $propagators)
+    {
+        $this->propagators = $propagators;
+        $this->fields = array_values(array_unique(array_merge(...array_map(static fn (TextMapPropagatorInterface $propagator) => $propagator->fields(), $propagators))));
```

Comment:
This line reads very heavy to me.  Do you think it mike make sense to break it up a bit?

Reply:
Gotta love PHP :sweat_smile:.  I'm open to alternative suggestions on how to get an array of unique values after concatenating a series of arrays.

I can move it to a private method.

Reply:
I may just be nitpicking here.  Fine to leave it as is.